src/screens/FeedScreen/RenderList.js

```diff
@@ -15,7 +15,9 @@ import {
   POST_TYPE_STANDARD,
   SOURCE_FEED_TAB
 } from '../../utils/constants';
-import dimen from '../../utils/dimen';
+import {Footer, Gap, PreviewComment} from '../../components';
+import {colors} from '../../utils/colors';
+import {getCommentLength} from '../../utils/getstream';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `dimen` has been removed but it's still being used in the code at line 116. This will cause a reference error. Please add the import statement back or replace `dimen` with an equivalent value.
```diff
- import dimen from '../../utils/dimen';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->